medusa/browser.py

```diff
@@ -112,7 +97,7 @@ def list_folders(path, include_parent=False, include_files=False):
     try:
         file_list = get_file_list(path, include_files)
     except OSError as e:
-        logger.log('Unable to open %s: %s / %s' % (path, repr(e), str(e)), logger.WARNING)
+        log.warning('Unable to open %s: %s / %s', (path, repr(e), str(e)))
```

Comment:
Shouldn't this be rewritten to the default logging format? Like '%s %s %s', value 1, value2, value3